docs/concepts/projects/dependencies.md

```diff
@@ -613,19 +613,47 @@ instead. Additionally, the `dev` group is [synced by default](#default-groups).
 
 ### Dependency groups
 
+uv supports
+[PyPA spec Dependency Groups](https://packaging.python.org/en/latest/specifications/dependency-groups/):
```

Comment:
Uncertain this is the "right" thing to cite? (genuine question, need to check with someone else...)

Reply:
The docs already reference [PEP 735](https://peps.python.org/pep-0735/).  I was going to reference it again but noticed that the pep itself points to the dependency groups spec as the canonical version.  So I linked to it.  

However, upon reviewing the docs, I realized there was further room for improvement so pushed up another commit with some additional refactoring which removes this sentence.

However, it's up for debate on whether or not the existing PEP 735 reference should be kept or replaced with the link to the spec.  I'd be +0 towards the latter since the PEP points there.